src/components/Forms/Subscribe.js

```diff
@@ -16,98 +16,140 @@ const SubscribeSchema = Yup.object().shape({
     .required('Required'),
 })
 
-const SignUp = ({ subscribed = false }) => (
-  <div>
-    <h2>Join the Newsletter</h2>
-    <Formik
-      initialValues={{
-        email_address: '',
-        first_name: '',
-      }}
-      validationSchema={SubscribeSchema}
-      onSubmit={values => {
-        // setTimeout(() => {
-        //   alert(JSON.stringify(values, null, 2))
-        // }, 500)
-        fetch('https://app.convertkit.com/forms/834199/subscriptions', {
+const PostSubmissionMessage = ({ response }) => {
+  console.log('Post submission response: ', response)
+
+  return <div>You have submitted with a status of {response.status}</div>
```

Comment:
Should this should be replaced with a nice looking "Thank you" component?

Probably don't need to display the response at all, but it's there for now